mods/ra/rules/vehicles.yaml

```diff
@@ -563,7 +563,7 @@ TTNK:
 	Armament:
 		Weapon: TTankZap
 		LocalOffset: 0,0,213
-	AttackFrontal:
+	AttackOmni:
```

Comment:
Thanks for fixing this; I also had the same issue with AttackCharge (which implements AttackOmni)